tools/compile_tool/README.md

```diff
@@ -44,24 +43,31 @@ compile_tool [OPTIONS]
                                              Notice that quotes are required.
                                              Overwrites precision from ip and op options for specified layers.
     -il                          <value>     Optional. Specifies layout for all input layers of the network.
-    -ol                          <value>     Optional. Specifies layout for all input layers of the network.
+    -ol                          <value>     Optional. Specifies layout for all output layers of the network.
     -iol                        "<value>"    Optional. Specifies layout for input and output layers by name.
                                              Example: -iol "input:NCHW, output:NHWC".
                                              Notice that quotes are required.
                                              Overwrites layout from il and ol options for specified layers.
+    -iml                         <value>     Optional. Specifies model layout for all input layers of the network.
+    -oml                         <value>     Optional. Specifies model layout for all output layers of the network.
+    -ioml                       "<value>"    Optional. Specifies model layout for input and output tensors by name.
+                                             Example: -ionl "input:NCHW, output:NHWC".
+                                             Notice that quotes are required.
+                                             Overwrites layout from il and ol options for specified layers.
+    -legacy                     "<value>"    Optional. Compile model to legacy format (layouts, data elements and
```

Comment:
something like `Compile model to legacy format for usage in Inference Engine API, by default compiles to OV 2.0 API` ?